File: src/test/java/com/love/StudentTest.java
package com.love;

import com.love.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void javaHighestTest(){
        Student student=new Student("West",60,80);
        Assertions.assertEquals(80,student.highest());
    }
    @Test
    public void javaGetAverage(){
        Student student=new Student("West",60,80);
        Assertions.assertEquals((60+80)/2,student.getAverage());
    }
}

File: src/test/kotlin/com/love/StudentTest.kt
package com.love.kotlin

import com.love.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest{
    @Test
    fun kotlinHighestTest(){
        val student=Student("West",60,80)
        Assertions.assertEquals(80,student.highest())
    }
    @Test
    fun kotlinGetAverageTest(){
        val student=Student("West",60,80)
        Assertions.assertEquals((60+80)/2,student.getAverage())
    }
    @Test
    fun kotlinGradingTest(){
        val student=Student("West",60,80)
        Assertions.assertEquals('C',student.grading())
    }
}

